.gitignore

```diff
@@ -6,6 +6,5 @@ nohup.out
 error.log
 *.idea
 *.csv
-*.sqlite3
```

Comment:
sqlite should be kept out of git.

Reply:
I am sorry。 I wanted to update the code, but accidentally carried out pull requests, suddenly be at a loss what to do. I don't understand what PR means. I'll pay attention to it later. I'm sorry